src/data/lock/lockfile_command.rs

```diff
@@ -14,17 +15,22 @@ pub struct LockfileCommand {
 impl<'a> LockfileCommand {
     pub fn from_command(
         local_package_name: &str,
-        local_package_version: &str,
+        local_package_version: Version,
         command: &'a Command,
     ) -> Self {
         // split the "package" field of the command if it exists
         // otherwise assume that this is a command for a local module
         // extract the package name and version for this command and insert into the lockfile command
-        let (package_name, package_version): (&str, &str) = match &command.package {
+        let (package_name, package_version): (&str, Version) = match &command.package {
             Some(package_string) => {
                 let split = package_string.as_str().split(' ').collect::<Vec<_>>();
                 match &split[..] {
-                    [package_name, package_version] => (package_name, package_version),
+                    [package_name, package_version] => {
+                        // this string must be parsed again because the package field on a command is a concatenated string
+                        // e.g. "_/pkg 1.0.0"
+                        let package_version = Version::parse(package_version).unwrap();
```

Comment:
`unwrap`

this can fail whenever a user mistypes a version, right?   We should have a proper error case here